demos/spring-5-demo/build.gradle

```diff
@@ -11,14 +11,14 @@ mainClassName = 'Main'
 
 repositories {
     mavenCentral()
-    maven { url "https://oss.sonatype.org/content/repositories/releases" }
+    mavenLocal()
```

Comment:
This forces the user to install service-validator's SNAPSHOT locally before trying out the demo, doesn't it? I think a better approach would be to deploy the snapshot and keep the repo reference remote? What are your thoughts on this?

Reply:
I agree, but this a simple example. I don't see any problem.

Reply:
Ok, fair enough.